website/docs/cli/install/apt.mdx

```diff
@@ -27,34 +27,44 @@ package repositories.
 
 ## Repository Configuration
 
-The Terraform packages are signed using a private key controlled by HashiCorp,
-so in most situations the first step would be to configure your system to trust
-that HashiCorp key for package authentication. For example:
+The Terraform packages are signed using a private key controlled by HashiCorp, so you must configure your system to trust that HashiCorp key for package authentication.
 
-```bash
-wget -qO - terraform.gpg https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /usr/share/keyrings/terraform-archive-keyring.gpg
-```
+To configure your repository:
 
-After registering the key, you can add the official HashiCorp repository to
-your system:
+1. Download the signing key to a new keyring.
 
-```bash
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/terraform-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/terraform.list
-```
+   ```bash
+   $ wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | \
+       sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg
+   ```
 
-The above command line uses the following sub-shell commands:
+1. Verify the key's fingerprint.
 
-* `dpkg --print-architecture` to determine your system's primary APT
-  architecture/ABI, such as `amd64`.
-* `lsb_release -cs` to find the distribution release codename for your current
-  system, such as `buster`, `groovy`, or `sid`.
+   ```bash
+   $ gpg --no-default-keyring --keyring /usr/share/keyrings/hashicorp-archive-keyring.gpg \
+       --fingerprint
+   ```
```

Comment:
```suggestion
   ```bash
   $ gpg --no-default-keyring \
       --keyring /usr/share/keyrings/hashicorp-archive-keyring.gpg \
       --fingerprint
   ```
```
Similar to above, give each argument its own line.